network/network.go

```diff
@@ -148,33 +140,46 @@ func (n *Network) runOnce(extIface *backend.ExternalInterface, inited func(bn ba
 
 	defer wg.Wait()
 
-	dur := n.bn.Lease().Expiration.Sub(time.Now()) - renewMargin
-
-	for {
-		select {
-		case <-time.After(dur):
-			err := n.sm.RenewLease(n.ctx, n.Name, n.bn.Lease())
-			if err != nil {
-				log.Error("Error renewing lease (trying again in 1 min): ", err)
-				dur = time.Minute
-				continue
-			}
-
-			log.Info("Lease renewed, new expiration: ", n.bn.Lease().Expiration)
-			dur = n.bn.Lease().Expiration.Sub(time.Now()) - renewMargin
-
-		case e := <-evts:
-			if e.Type == subnet.EventRemoved {
-				log.Warning("Lease has been revoked")
-				interruptFunc()
-				return errInterrupted
+	if n.bn.Lease() != nil {
```

Comment:
This one is hard to swallow as it makes it unclear what a network does if not for getting itself a lease for this node. I looked at your ovs branch and I see how you're using it but I feel uneasy merging this change. In essence the only thing that the network does in ovs branch is read in the config with VNI from etcd and output that VNI to the subnet.env file. I feel that if you'd like to use one big address space for all the networks, it's easier to have flannel just set up one network and do flow management (with VNI tagging) completely out of band from flannel. You can still store tenant->VNI mapping somewhere in etcd.